components/admin_console/data_retention_settings/data_retention_settings.scss

```diff
@@ -21,4 +21,19 @@
     .actionIcon {
         flex-grow: 0.15;
     }
+
+    .dropdown-menu {
+        background: var(--sys-center-channel-bg);
+
+        > li > button:hover {
+            background: rgba(var(--sys-center-channel-color-rgb), 0.1);
+        }
+    }
+
+    input:-webkit-autofill,
+    input:-webkit-autofill:hover,
+    input:-webkit-autofill:focus,
+    input:-webkit-autofill:active {
+        -webkit-box-shadow: 0 0 0 30px var(--sys-center-channel-bg) inset;
+    }
```

Comment:
@martinkraft or @BenCookie95 , this is an attempt to fix the box-shadow issue, but it doesn't work because it's being overwritten by a `!important` attribute.

I found this bit of code in `mattermost-webapp/components/payment_form/payment_form.scss`:
```
input:-webkit-autofill,
input:-webkit-autofill:hover,
input:-webkit-autofill:focus,
input:-webkit-autofill:active {
    -webkit-box-shadow: 0 0 0 30px var(--center-channel-bg) inset !important;
}
```

As far as I can tell, this is the only `.scss` file that changes a `-webkit-box-shadow`, and the `!important` overwrites the styling I tried. I'm not sure if it would be better to remove the `!important` or find some other way to override this.

Reply:
That style shouldn't be at the root of that scss file, it will impact every input in the application I think. You should move that block inside the `.PaymentForm` block

Reply:
Done! Let me know if that works, please